src/client/core.ts

```diff
@@ -83,7 +86,7 @@ export async function aptosRequest<Req extends {}, Res extends {}>(
 
   // to support both fullnode and indexer responses,
   // check if it is an indexer query, and adjust response.data
-  if (aptosConfig.isIndexerRequest(url)) {
+  if (type === AptosApiType.INDEXER) {
```

Comment:
hmm not sure we can assume that a custom url/service acts the same as our own service so it makes sense to check if it is an aptos indexer response and then do the conversation - we might need to do a different logic for custom services but not sure how custom indexer service is common

Reply:
I think at this point we assume it works the same, given that we assume all the tables that exist and all.

Any custom indexing at this point basically would need its own client, separate of the SDK